### DISCUSSION TITLE
v0.1.0-alpha.6

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#31] [Releases]
## What's Changed
* Filament user trait and result stats error by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/30

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.1.0-alpha.5...v0.1.0-alpha.6

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.1.0-alpha.6'>v0.1.0-alpha.6</a>.</em>

**Comment by jiriteach** [comment]
Hi - Any chance of a build for linux/arm64/v8 please? Want to setup and run on my Raspberry Pi 4. Thanks

**Reply by alexjustesen** [reply]
On the roadmap, I've got a k3s cluster on some rpi's I'll run this on so definitely going to happen soon as I can.

**Reply by jiriteach** [reply]
Awesome - thanks! Really keen on testing when available.

**Reply by alexjustesen** [reply]
Should you want to follow progress on multi-arch support I opened an issue and will link to any PR's there https://github.com/alexjustesen/speedtest-tracker/issues/32
